src/mlpack/tests/activation_functions_test.cpp

```diff
@@ -1241,3 +1242,21 @@ TEST_CASE("TanhExpFunctionTest", "[ActivationFunctionsTest]")
   CheckActivationCorrect<TanhExpFunction>(activationData, desiredActivations);
   CheckDerivativeCorrect<TanhExpFunction>(desiredActivations, desiredDerivatives);
 }
+
+TEST_CASE("SILUFunctionTest","[ActivationFunctionsTest]") {
+    //Random generated values
+    const arma::colvec activationData("-2 2 4.5 -5.7 -1 1 0 10");
+
+    //Calculated
```

Comment:
We might want to be more specific here, I guess the values are hand calculated, or from another framework?